src/ScaleUnitManagement/UserConfig.sample.xml

```diff
@@ -23,6 +23,12 @@
     <!-- If empty, the existing value in web.config will be used - otherwise the web.config will get overwritten with this value. -->
     <AADTenantId></AADTenantId>
   </AADConfiguration>
+
```

Comment:
There is a comment below that explains the different types, can we add the new type there as well?